kvmagent/kvmagent/test/network_plugin_testsuite/test_bridge_api.py

```
@@ -48,8 +48,8 @@ def test_create_bridge(self):
         network_plugin_utils.delete_novlan_bridge(bridgeName=br_name, physicalInterfaceName=interF)
 
     @pytest_utils.ztest_decorater
-    def test_add_if_to_bridge(self):
-        r, o = bash.bash_ro("ip a| grep BROADCAST|grep -v virbr | awk -F ':' 'NR==2{print $2}' | sed 's/ //g'")
+    def test_d_add_if_to_bridge(self):
+        r, o = bash.bash_ro("ip a| grep BROADCAST|grep -v virbr | awk -F ':' 'NR==1{print $2}' | sed 's/ //g'")
```

Ruff (0.15.2)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)